Raids/Prophecy of Ahnket/LFR/Tartaric_Depths/Beligosh.lua

```diff
@@ -107,14 +110,13 @@ BEL.Lang.Buff = {}
 -- Debuff Dictionary
 BEL.Lang.Debuff = {}
 
-BEL.Lang.Debuff.VengeanceOfBeligosh = KBM.Language:Add("Vengeance of Beligosh")--tank debuff
-BEL.Lang.Debuff.VengeanceOfBeligosh:SetGerman("Rache von Beligosh")
-BEL.Lang.Debuff.VengeanceOfBeligosh:SetFrench("Vengeance de Beligosh")
-
-BEL.Lang.Debuff.SeedOfImmolation = KBM.Language:Add("Seed of Immolation")--raid debuff
-BEL.Lang.Debuff.SeedOfImmolation:SetGerman("Saat der Opferung")
-BEL.Lang.Debuff.SeedOfImmolation:SetFrench("Graine d'immolation")
+BEL.Lang.Debuff.Vengeance = KBM.Language:Add("Vengeance of Beligosh")--tank debuff
```

Comment:
Why has the variables names been changed? 
(VenganceOfBligosh and SeedOfImmolation)